CHANGELOG.md

```diff
@@ -1,7 +1,71 @@
+ - Fixed screen reader can't identify extension button. ([#8269](https://github.com/brave/browser-laptop/issues/8269))
+ - Fixed tabs bar size by another 2 px based on increased user feedback. ([#8263](https://github.com/brave/browser-laptop/issues/8263))
+ - Fixed memory leak when opening then closing a lot of tabs rapidly. ([#8244](https://github.com/brave/browser-laptop/issues/8244))
+ - Fixed about:brave copy to clipboard tooltip. ([#8199](https://github.com/brave/browser-laptop/issues/8199))
+ - Fixed wallet creation failure. ([#8195](https://github.com/brave/browser-laptop/issues/8195))
+ - Fixed invisible pinned tab after upgrade. ([#8190](https://github.com/brave/browser-laptop/issues/8190))
+ - Fixed URL bar shape consistency when payments are enabled/disabled. ([#8170](https://github.com/brave/browser-laptop/issues/8170))
+ - Fixed creating a bookmark by dragging the Url bar icon. ([#8151](https://github.com/brave/browser-laptop/issues/8151))
+ - Fixed save .torrent file action. ([#8146](https://github.com/brave/browser-laptop/issues/8146))
+ - Fixed image title. ([#8116](https://github.com/brave/browser-laptop/issues/8116))
+ - Fixed white tab text on dark tab background which is difficult to read. ([#8115](https://github.com/brave/browser-laptop/issues/8115))
+ - Fixed javascript in bookmarks cause connection info box to appear. ([#8087](https://github.com/brave/browser-laptop/issues/8087))
+ - Fixed LastPass opening in the wrong position when activated by keyboard. ([#8034](https://github.com/brave/browser-laptop/issues/8034))
+ - Fixed about pages cannot be synced because data is too big. ([#8023](https://github.com/brave/browser-laptop/issues/8023))
+ - Fixed toggle excluding a site from ledger sometimes removes it from the list. ([#7987](https://github.com/brave/browser-laptop/issues/7987))
+ - Fixed Sync does not appear to recover from network outage. ([#7972](https://github.com/brave/browser-laptop/issues/7972))
+ - Fixed Sync losing hierarchy when adding bookmarks to new sync members. ([#7971](https://github.com/brave/browser-laptop/issues/7971))
+ - Fixed localization ability for delete confirmation message. ([#7958](https://github.com/brave/browser-laptop/issues/7958))
+ - Fixed default browser setting Ubuntu. ([#7800](https://github.com/brave/browser-laptop/issues/7800))
+ - Fixed dead tab when enter is pressed twice after typing in URL. ([#7727](https://github.com/brave/browser-laptop/issues/7727))
+ - Fixed view source shortcut to be CTRL+U, not CTRL+ALT+U. ([#7702](https://github.com/brave/browser-laptop/issues/7702))
+ - Fixed submenu position for bookmarks. ([#7662](https://github.com/brave/browser-laptop/issues/7662))
+ - Fixed right click not dismissing / hiding an open menu. ([#7403](https://github.com/brave/browser-laptop/issues/7403))
+ - Fixed default browser setting reset after upgrade. ([#5246](https://github.com/brave/browser-laptop/issues/5246))
```

Comment:
I would vote to move this to the very top since it has a massive impact; it should also include `Windows`, like:
`Fixed issue where default browser was reset after upgrade on Windows`